_includes/admin-domains.md

```diff
@@ -16,10 +16,18 @@
 
 1. Sign in to {{ product_link }}{: target="_blank" rel="noopener" class="_"}.
 2. {{ domain_navigation }}
-3. Select **Add a domain** and continue with the on-screen instructions to add the TXT Record Value to your domain name system (DNS).
+3. Select **Add a domain**.
+4. Continue with the on-screen instructions to get the **TXT Record Value**. Copy this value to paste into a TXT record that you create in your Domain Name System (DNS) settings.
 
     >**Note**
     >
     > Format your domains without protocol or www information, for example, `yourcompany.example`. This should include all email domains and subdomains users will use to access Docker, for example `yourcompany.example` and `us.yourcompany.example`. Public domains such as `gmail.com`, `outlook.com`, etc. aren’t permitted. Also, the email domain should be set as the primary email.
 
-4. Once you have waited 72 hours for the TXT Record verification, you can then select **Verify** next to the domain you've added, and follow the on-screen instructions.
\ No newline at end of file
+5. Add a TXT record to your DNS settings. The steps to do so may vary depending on your DNS provider.
+
+   >**Note**
+   >
+   > When you add the TXT record, ensure that you create the record on the root of the zone. This is typically denoted with the `@` name for the record.
```

Comment:
@ivan-californias Just wanted to verify with you that this wording looks good since this seemed to be the area some users were struggling with based on some Slack threads where you found the record wasn't created on the root.

Reply:
**suggestion:** There are some cases where the customer might not want to use the root of their domain, ie `us.yourcompany.com`. I'd rephrase this block to something similar to:

> Make sure that TXT record name that you create on your DNS matches with the domain you registered on Docker (Step4). For example, if you registered the subdomain `us.yourcompany.com` you need to create a TXT record within the same name/zone `us`. If you registered a root domain `yourcompany.com` you need to create a TXT record on the root zone, which is typically denoted with the `@` as name for the record.

Reply:
Sounds good! Will update.